### DISCUSSION TITLE
🔥 FrontendBR Space • Front in Sampa 2019

### DISCUSSION THREAD

**Discussion by felipefialho** [frontendbr/forum#1470] [Dicas]
![image](https://user-images.githubusercontent.com/3603793/60529300-b1195680-9ccc-11e9-95db-7d8932a0313a.png)

Depois do sucesso na [última edição do Front in Sampa](https://github.com/frontendbr/forum/issues/1124) e no [Front in Campinas](https://github.com/frontendbr/forum/issues/1384), é com muito orgulho que oficializamos mais uma edição **FrontendBR Space**! 

**FrontendBR Space** é um espaço aberto e pensado para que todos vocês: 

- Relaxem
- Troquem idéia 
- Conheçam pessoas
- Aprendam
- Ensinem

E principalmente se divirtam ainda mais durante o evento 😄  

Novamente vai acontecer no [Front in Sampa](https://www.frontinsampa.com.br/) que acontece no dia 13 de Julho de 2019 em São Paulo.

Além disso vamos ter camisetas exclusivas que vão ser sorteadas durante o evento. Sensacional né? 😍

![image](https://user-images.githubusercontent.com/3603793/60529681-7d8afc00-9ccd-11e9-825b-73526a9019d1.png)

Esse é um presente do Front in Sampa e do Front-end BR  ❤️

**Comment by phhoffmann** [comment]
@felipefialho alguma chance de podermos comprar as camisetas? :)

**Comment by Kamilahsantos** [comment]
que camiseta linda <3

**Comment by kvnol** [comment]
Não vou conseguir ir ao frontinsampa, mas tô apaixonado nessa camiseta.

**Comment by notnotgabriel** [comment]
Eu digo bora!

**Comment by felipefialho** [comment]
@phhoffmann A princípio não tem como :(

Quem sabe no futuro?

**Comment by phhoffmann** [comment]
por favor, mano! bota meu nome em primeiro na lista aí <3

**Comment by MrS4w** [comment]
Só vamo

**Comment by thainamonteiro** [comment]
Que camisa linda ♥

**Comment by felipefialho** [comment]
@thainamonteiro Ficou foda demais mesmo 😄

**Comment by MRRaduan** [comment]
Eu comprei ingresso para o Front in Sampa mas não vou conseguir estar presente.
Agora o arrependimento de não poder ir aumentou mais ainda. 

;-;

Alguém interessado em comprar meu ingresso? Vendo pelo preço do primeiro lote.

**Comment by ninetails** [comment]
se eu tivesse lido antes :( #fuu

<sup>(nem vou poder ir tb :\ )</sup>

**Comment by felipefialho** [comment]
Valeu todo mundo que colou 😄

**Comment by Kamilahsantos** [comment]
Adorei o espaço e adorei a camiseta tbm que ganhei, maravitop <3

**Comment by phhoffmann** [comment]
valeu mano! foi um prazer te conhecer e trocar uma idéia. agora só falta abrir as camisetas pra venda kkk
